src/components/Form/index.tsx

```diff
@@ -164,7 +164,7 @@ class Form extends React.Component<FormProps, FormState> {
             event.preventDefault();
           }
         });
-      } else if (child.type === SelectField) {
+      } else if (child.type === SelectField || Select) {
```

Comment:
Probably mistake, and should be sth like `[SelectField, Select].includes(child.type)`

Reply:
fixed